core/src/main/scala/org/apache/spark/rdd/DoubleRDDFunctions.scala

```diff
@@ -152,10 +152,12 @@ class DoubleRDDFunctions(self: RDD[Double]) extends Logging with Serializable {
 
   /**
    * Compute a histogram using the provided buckets. The buckets are all open
-   * to the right except for the last which is closed
+   * to the right except for the last which is closed.
+   * {{{
    *  e.g. for the array
    *  [1, 10, 20, 50] the buckets are [1, 10) [10, 20) [20, 50]
    *  e.g 1<=x<10 , 10<=x<20, 20<=x<=50
+   * }}}
```

Comment:
I just wanted to avoid to use inlined tags simply because my IDE shows some warnings..
<img width="266" alt="2016-11-27 11 41 53" src="https://cloud.githubusercontent.com/assets/6477701/20649122/1e11c98c-b4fb-11e6-9e6c-af048d12f0e8.png">

This seems a valid javadoc markdown but it seems any inlined tags are treated as backticts in scaladoc.

(I am sorry for messing around with many comments but this is also related with https://github.com/apache/spark/pull/16013#discussion_r89666914)